test/common/filesystem/filesystem_impl_test.cc

```diff
@@ -12,6 +12,8 @@
 namespace Envoy {
 namespace Filesystem {
 
+static constexpr FlagSet DEFAULT_FLAGS{0xf};
```

Comment:
You can use the other default you already defined? Same elsewhere.

Reply:
Use static flagset .

Reply:
I'd use DefaultFlags for the spelling of this constant, as DEFAULT_FLAGS might clash with a macro in some header somewhere, and macros are namespace-blind.

Reply:
thanks and will change.